cachi2/core/package_managers/npm.py

```diff
@@ -87,6 +104,11 @@ def semver_version(self) -> Optional[str]:
         else:
             return None
 
+    @property
+    def dependencies(self) -> Optional[Any]:
+        """Get the dict of package dependencies, if available."""
+        return self._package_dict.get("dependencies") or None
```

Comment:
This line is not covered, which probably means we don't use the property at all?

Reply:
It seems so, I'd say remove it.

Reply:
Nice catch, removed